.github/workflows/build-image-reusable.yaml

```diff
@@ -0,0 +1,60 @@
+name: Build unified Docker image
+
+on:
+    workflow_call:
+        inputs:
+            if:
+                description: 'Whether to run this job'
+                required: false
+                default: true
+                type: boolean
+            name:
+                required: true
+                type: string
+            key_secret_sentry:
```

Comment:
This reads oddly to me.

`sentry_secret_key`
`posthog_secret_key`

Reply:
it's because it's not the key itself but the key of `secrets` object. I'll rename to something more explicit